app/views/issues/_attributes.rhtml

```diff
@@ -1091,6 +1111,12 @@ Event.observe(window, 'load', function() {
         }
     }
 
+    var partsStatus = partsStatusInput.options[this.selectedIndex].label
+    if (statusInput.value = 5 && partsStatus == 'Pending') {
+      updateReturnVisitAndWaitingParts("NO")
+    }
+
+    updateFieldVisits()
```

Comment:
This should get called whenever one of the dispatch fields changes, not just on page load.

Reply:
I'll remove it, because it's a part of the other jira

Reply:
> This should get called whenever one of the dispatch fields changes, not just on page load.

it's kinda hard, since there are 7 fields for dispatches, it's easier to update it in the submit, even though it's not good UX

Reply:
Ah, I didn't realize this was part of submit, that works then.